packages/bsky/proto/bsky.proto

```diff
@@ -969,13 +969,21 @@ message GetRecordTakedownResponse {
 
 // Polo-backed Graph Endpoints
 
-// GetFollowsFollowing gets the list of DIDs that the actor follows that also follow the target
+
```

Comment:
I think we ended up accumulating some more protos here than we needed

Reply:
also I think we need a `pnpm format` to clean up some of the buf codegen as well